src/shared/components/annotation/OncoKB.tsx

```diff
@@ -136,6 +135,23 @@ export default class OncoKB extends React.Component<IOncoKbProps, {}>
         );
     }
 
+    public errorIcon()
+    {
+        return (
+            <DefaultTooltip
+                overlay={<span>Error fetching OncoKB data</span>}
+                placement="right"
+                trigger={['hover', 'focus']}
+                arrowContent={<div className="rc-tooltip-arrow-inner"/>}
```

Comment:
Good to know. Seems like it was added at a later stage in development. Initially we didn't have that prop. In this case, I think we still have some leftover default `arrowContent`s in other components, too...